### ISSUE TITLE
[FEATURE] Live view and recorded video options

### ISSUE THREAD

**Issue by spupuz** [roflcoopter/viseron#442]
- Add a view of all cameras live view
- Add an option to display OSD time on live view camera and on recorded videos
- Add an option to make possible on liveview to see the motiont detection (like in this linked picture)

https://hackster.imgix.net/uploads/attachments/1110817/_TqhCa15kFL.blob?auto=compress&w=1600&h=1200&fit=min&fm=jpg

**Comment by spupuz** [comment]
![image](https://user-images.githubusercontent.com/1721792/213200997-5574a4df-cc3d-4fba-bc21-39090c145272.png)

i would like a live view like this

**Comment by parish99** [comment]
I agree this would be a nice addition to the software.  It would aslo be cool if there was a way to show multiple live cameras in a single stream.

**Comment by roflcoopter** [comment]
Long overdue, but this is available now in `dev` and will be in the next release
